script/bootstrap

```diff
@@ -12,6 +12,7 @@ SOURCE_ROOT = os.path.abspath(os.path.dirname(os.path.dirname(__file__)))
 def main():
   os.chdir(SOURCE_ROOT)
   result = (subprocess.call(['git', 'submodule', 'sync']) or
+            subprocess.call(['git', 'config', 'core.autocrlf', 'false']) or
```

Comment:
I don't think it could help, files in `patches/` dir will already be there when this command is executed.
Can we configure git on a build machine before cloning the libcc repo?